lightly/utils/debug.py

```diff
@@ -105,8 +106,7 @@ def generate_grid_of_augmented_images(
             )
     else:
         raise ValueError(
-            "Collate function must be one of "
-            "(BaseCollateFunction, MultiViewCollateFunction) "
+            "Collate function must be one of (BaseCollateFunction, MultiViewCollateFunction) "
```

Comment:
The line length should be shorter than 80 chars, see the [google style guide](https://google.github.io/styleguide/pyguide.html#32-line-length)